tests/test_demo.py

```
@@ -0,0 +1,115 @@
+from snakesee.demo.runner import _build_snakemake_argv
+from snakesee.demo.runner import _cache_root
+from snakesee.demo.runner import _copy_workflow
+from snakesee.demo.runner import run_demo
+
+
+class TestSnakefileDryRun:
+    def test_dry_run_succeeds(self, tmp_path: Path) -> None:
+        """The bundled Snakefile parses cleanly and resolves the DAG."""
+        _copy_workflow(tmp_path)
+        result = subprocess.run(
+            ["snakemake", "--dry-run", "all"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)